server/sonar-server/src/test/java/org/sonar/server/computation/queue/ReportSubmitterTest.java

```diff
@@ -194,7 +203,7 @@ public void submit_a_report_on_new_project_with_global_scan_permission() {
   }
 
   @Test
-  public void submit_a_report_on_existing_project_with_global_scan_permission() {
+  public void submit_a_report_on_existing_project_with_scan_permission_on_organization() {
     userSession.setGlobalPermissions(SCAN_EXECUTION);
```

Comment:
Note that the goal is to fully drop `setGlobalPermissions()` and `hasPermission()`.